pkg/probe/probes.go

```diff
@@ -176,50 +178,41 @@ func runPing(_ client.Client, timeout time.Duration) error {
 	}
 	return nil
 }
-func lookupRootNS(nameServer string, timeout time.Duration) error {
-	rootNS := "root-server.net"
-	r := &net.Resolver{
-		PreferGo: true,
-		Dial: func(ctx context.Context, network, address string) (net.Conn, error) {
-			d := net.Dialer{
-				Timeout: timeout,
-			}
-			return d.DialContext(ctx, network, net.JoinHostPort(nameServer, "53"))
-		},
-	}
-	// We use a closure to create a scope for defer here
-	ctx, cancel := context.WithTimeout(context.Background(), timeout)
-	defer cancel()
-	_, err := r.LookupNS(ctx, rootNS)
-	if err != nil {
-		return errors.Wrapf(err, "failed looking up NS %s using name sever %s", rootNS, nameServer)
-	}
-	return nil
-}
 
-func runDNS(_ client.Client, _ time.Duration) error {
+func runDNS(_ client.Client, timeout time.Duration) error {
+	// Get the name servers at node since the ones at container are not accurate
 	currentStateAsGJson, err := currentStateAsGJson()
```

Comment:
If we don't wrap `currentStateAsGJson` we are doing nothing since is the one retrieving currentState.

Reply:
Right, thanks!